spec/adapterSpec.js

```diff
@@ -177,18 +189,17 @@ describe('webdriverJS Jasmine adapter', function() {
   });
 
   // Uncomment to see timeout failures.
-
-  // it('should timeout after 200ms', function() {
+  // it('should timeout after 200ms', function(done) {
   //   expect(fakeDriver.getValueA()).toEqual('a');
-  // }, 300);
-
+  // }, 200);
+  
   // it('should timeout after 300ms', function() {
   //   fakeDriver.sleep(9999);
   //   expect(fakeDriver.getValueB()).toEqual('b');
   // }, 300);
 
   // it('should pass errors from done callback', function(done) {
-  //   done('an error');
+  //   done.fail('an error');
   // });
 
   it('should pass after the timed out tests', function() {
```

Comment:
In my earlier PR working on a proto version of this (https://github.com/angular/jasminewd/pull/5), I added a describe block for tests that should fail. Can we add that to a new spec file and verify that things that should fail still fail? We can just run in manually and not put it in `npm test` or make a quick script around `npm test` that verifies that the fail count is correct.